### PR TITLE
fix(aiohttp): fixed decode error

### PR DESCRIPTION
**Pull request by osherv**

### CODE REVIEW

File: cisco_telescope/instrumentations/aiohttp/__init__.py [hunk 1]
```diff
@@ -176,17 +177,24 @@ async def on_request_chunk_sent(
...
-            decoded_chunk = params.chunk.decode()
-            trace_config_ctx.response_body += decoded_chunk
+            trace_config_ctx.response_body += _decode_chunk(params.chunk)
+
+    def _decode_chunk(chunk):
+        try:
+            decoded_chunk = chunk.decode()
+        except UnicodeDecodeError as e:
+            decoded_chunk = str(chunk)
+            logging.info(f"Could'nt decode the chunk: {decoded_chunk}, {e}")
```
Comment (nemoshlag) [inline cisco_telescope/instrumentations/aiohttp/__init__.py:195]: typo :) maybe "Could not decode body chunk"

File: tests/instrumentations/test_aiohttp.py [hunk 2]
```diff
@@ -68,6 +69,29 @@ async def test_post_request_sanity(self):
                 self.response_body(),
             )
 
+    async def test_post_request_invalid_utf8(self):
+        Configuration().payloads_enabled = True
+        data_to_send = b'/">Miso\xdfNoContinuation'
```
Comment (nemoshlag) [inline tests/instrumentations/test_aiohttp.py:74]: awesome, maybe just add something to verify the info log created to verify this flow being tested

File: cisco_telescope/instrumentations/aiohttp/__init__.py (hunk 1 above)
Comment (osherv) [reply cisco_telescope/instrumentations/aiohttp/__init__.py:195]: fixed

File: tests/instrumentations/test_aiohttp.py (hunk 2 above)
Comment (osherv) [reply tests/instrumentations/test_aiohttp.py:74]: fixed

**Review by nemoshlag — APPROVED** [review]

**Pull request merged**